src/main/src/main/java/org/geoserver/catalog/impl/MapInfoImpl.java

```diff
@@ -11,6 +11,7 @@
 import org.geoserver.catalog.LayerInfo;
 import org.geoserver.catalog.MapInfo;
 
+@SuppressWarnings("serial")
```

Comment:
I would tell the IDE to ignore serial version UID warnings... none of this classes is meant to be serialized and re-used long term, and very often we change the structure of the class without changing the serialVersionUID anyways, which would break its intended usage anyways.